Libraries/Components/WebView/WebView.ios.js

```diff
@@ -213,6 +217,8 @@ var RCTWebView = createReactIOSNativeComponentClass({
   validAttributes: merge(ReactIOSViewAttributes.UIView, {
     url: true,
     html: true,
+    allowBounce: true,
```

Comment:
can you use the same name as <ScrollView> component? I think it's just `bounces` and `scrollEnabled`